app/konnect/getting-started/index.md

```diff
@@ -10,22 +10,22 @@ New to {{site.konnect_saas}}? Get started with the basics through the web app:
 
     [Create an account &gt;](/konnect/getting-started/access-account/)
 
-1.  **Set up a runtime instance**:
+1.  **Set up a data plane node**:
 
-    Set up your first runtime instance, which gives you a data plane for proxying traffic.
+    Set up your first data plane node, which gives you a data plane for proxying traffic.
 
     Start with a data plane proxy so that when your service configuration is
     ready, your services are immediately connected, configured,
     and available for testing.
 
-    [Set up a runtime instance &gt;](/konnect/getting-started/configure-runtime/)
+    [Set up a data plane node &gt;](/konnect/getting-started/configure-data-plane-node/)
```

Comment:
```suggestion
    [Create a data plane node &gt;](/konnect/getting-started/configure-data-plane-node/)
```

I think configure  or connect could also work here, it depends on your perspective. The node is already created you run a script on it and it's configured to connect and work on Konnect. 
set up works better with something ephemeral like runtime instance.